### PR TITLE
fixes issue #34 - put on hold

### PR DESCRIPTION
**Pull request by ClaudiaFrank**

### CODE REVIEW

**Comment by ClaudiaFrank** [comment]
While it solves the creation of scripts issue it generates display issues of menu items.
On hold until solved.

**Comment by chcg** [comment]
@ClaudiaFrank Do you mean the issue from #20:

> Could you add Unicode support to the menu?
> Hebrew file-names are displayed as gibberish.

?

File: PythonScript/src/PythonScript.cpp [hunk 1]
```diff
@@ -621,43 +621,37 @@ static void ensurePathExists(const tstring& path)
 
 static void newScript()
 {
-	
-	OPENFILENAMEA ofn;
-	memset(&ofn, 0, sizeof(OPENFILENAMEA));
+	OPENFILENAMEW ofn;
```
Comment (chcg) [inline PythonScript/src/PythonScript.cpp]: Should be possible to use the version OPENFILENAME, GetSaveFileName, CreateFile without the W extension as we are building with Unicode and in this case it is anyhow mapped to the W versions, see e.g. docu of https://msdn.microsoft.com/en-us/library/windows/desktop/ms646839(v=vs.85).aspx
at the end of the page in requirements

**Comment by ClaudiaFrank** [comment]
Kind of yes, it is a little bit strange, it does work under subdirectories and in main menu when using
Configuration tool (see script ÜberÄÖß.py)

https://user-images.githubusercontent.com/16170444/39311440-3cc21348-496d-11e8-8c89-7bb3ce65f011.png

but running it by clicking the item under subdirectory corrupts the 
Run Previous Script menu item

https://user-images.githubusercontent.com/16170444/39311612-b22cb796-496d-11e8-827e-cd097a6548df.png

and clicking the ÜberÄÖß.py under main menu does NOT corrupt the Run Previous Script menuitem
BUT does not execute the script at all.

**Comment by chcg** [comment]
Seems to be caused by:
 void MenuManager::updatePreviousScript(const char *filename)

**Comment by ClaudiaFrank** [comment]
I guess/assume I understood what the problem is.
PythonScript does, internally, char to wchar/wchar to char conversion by using utf-8.
I assume this is related to being pythons "native" encoding and consoles output encoding.
Now, windows uses, afaik, ucs-2 and acp internally - so an additional WcharMbcsConverter
like let's say acp2wchar and/or wchar2acp should do the trick (at least I hope so)

I will follow up on this and see if this is really the case.

**Comment by ClaudiaFrank** [comment]
This is still not perfect, even so menu issues should have been solved another issue raised
You cannot use console.writeError function together with python byte strings

Left, the white background = current situation (without unicode path support)
Right, black background = the new situation

grafik

In case that one wants to use the extended chars to be displayed as "error text" it is needed
to use unicode strings. I haven't find a suitable way to get rid of this.
Any ideas are very much appreciated.

**Comment by chcg** [comment]
See PR #60, so this one could be closed, as discussed offline

**Pull request closed without merging**